api/types_test.go

```diff
@@ -0,0 +1,132 @@
+package api
+
+import (
+	"testing"
+)
+
+var (
+	OIDCNotEnabled     = false
+	OIDCIsEnabled      = true
+	emptyFIlename      = ""
+	OIDCConfigFilename = "./secrets/oidc-config.yaml.sample"
+)
+
+func TestOIDCConfig_Validate(t *testing.T) {
```

Comment:
As mentioned in `https://github.com/bf2fc6cc711aee1a0c2a/observability-remote-write-proxy/pull/12#discussion_r1114172741` Validate should be reserved to validation and reading of the file should be decoupled from it to make it more testable